src/components/ProgressBar.tsx

```diff
@@ -1,76 +1,65 @@
 import 'react-sweet-progress/lib/style.css';
-
-import React from 'react';
 // @ts-ignore
 import { Progress } from 'react-sweet-progress';
 import styled from '@emotion/styled';
+import { Flex, useMediaQuery } from '@chakra-ui/react';
 
 interface ProgressBarProps {
-  children: React.ReactNode;
+  kindOfDust: string;
   id: string;
   state: number;
 }
 
-const ProgressBar = ({ children, id, state }: ProgressBarProps) => (
-  <DustProgressWrapper id={id}>
-    <DustWrapperFlex>
-      <div>{children}</div>
-      <DustFigure>{state}</DustFigure>
-    </DustWrapperFlex>
-    <ProgressWrapper>
-      <Progress
-        percent={+state * (1 / 1000) * 100}
-        theme={{
-          success: {
-            symbol: ' ',
-            color: 'rgb(223, 105, 180)',
-          },
-          active: {
-            symbol: ' ',
-            color: '#fbc630',
-          },
-          default: {
-            symbol: ' ',
-            color: '#fbc630',
-          },
-        }}
-      />
-    </ProgressWrapper>
-  </DustProgressWrapper>
-);
-
-export default ProgressBar;
+const ProgressBar = ({ kindOfDust, id, state }: ProgressBarProps) => {
+  const percent = state * (1 / 100) * 100;
+  const percentOfDustContamination = percent > 100 ? 100 : percent;
+  const [isLargerThan768] = useMediaQuery('min-width: 768px');
```

Comment:
```suggestion
  const [isLargerThan768] = useMediaQuery('(min-width: 768px)');
```

공식문서랑 다르네요 https://chakra-ui.com/docs/hooks/use-media-query